.gitignore

```diff
@@ -0,0 +1 @@
+*node_modules/*
```

Comment:
`node_modules` should be enough 
https://stackoverflow.com/questions/29820791/git-ignore-node-modules-folder-everywhere